src/input_handler.hh

```diff
@@ -171,20 +173,24 @@ constexpr auto enum_desc(Meta::Type<AutoComplete>)
     });
 }
 
+bool should_show_info(AutoInfo mask, const Context& context);
 bool show_auto_info_ifn(StringView title, StringView info, AutoInfo mask, const Context& context);
 void hide_auto_info_ifn(const Context& context, bool hide);
 
 template<typename Cmd>
 void on_next_key_with_autoinfo(const Context& context, StringView mode_name,
                                KeymapMode keymap_mode, Cmd cmd,
-                               StringView title, StringView info)
+                               String title, String info)
 {
-    const bool hide = show_auto_info_ifn(title, info, AutoInfo::OnKey, context);
+    using namespace std::placeholders;
     context.input_handler().on_next_key(mode_name,
-        keymap_mode, [hide,cmd](Key key, Context& context) mutable {
+        keymap_mode, [cmd](Key key, Context& context) mutable {
+            bool hide = should_show_info(AutoInfo::OnKey, context);
             hide_auto_info_ifn(context, hide);
             cmd(key, context);
-    });
+    }, std::bind([&context](Timer&, String title, String info) {
+           show_auto_info_ifn(title, info, AutoInfo::OnKey, context);
+    }, _1, std::move(title), std::move(info)));
```

Comment:
std::bind on a lambda seems weird, why not capture title and info ?
```suggestion
    }, [&context, title=std::move(title),info=std::move(info)](Timer&) {
           show_auto_info_ifn(title, info, AutoInfo::OnKey, context);
    });
```

Reply:
Done